sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -2050,6 +2149,25 @@ public partial class PredictiveValue
         public System.DateTimeOffset TimeStamp { get { throw null; } }
         public double Value { get { throw null; } }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct PublicNetworkAccess : System.IEquatable<Azure.ResourceManager.Monitor.Models.PublicNetworkAccess>
```

Comment:
This name is too generic, please use `rename-mapping` to rename it. Maybe rename it to `MonitorPublicNetworkAccessState`?